src/datasets.js

```diff
@@ -272,7 +272,7 @@ const datasets = {
 					'line-width': highwayTypeLineWidths
 				}
 			},
-			'Traffic volume category': {
+			'Traffic volume category (pcu/day)': {
```

Comment:
I think it's annual average daily car traffic, we're not accounting for cars as the PCU formulae do: https://en.wikipedia.org/wiki/Passenger_car_equivalent

```suggestion
			'Traffic volume category (AADT)': {
```